Gemfile

```diff
@@ -196,6 +196,10 @@ group :ui_dependencies do # Added to Bundler.require in config/application.rb
   gem "jquery-rjs",                   "=0.1.1",                       :git => "https://github.com/ManageIQ/jquery-rjs.git", :tag => "v0.1.1-1"
 end
 
+group :v2v, :ui_dependencies do
+  gem "miq_v2v_ui", :git => "https://github.com/priley86/miq_v2v_ui_plugin.git", :branch => "master"
```

Comment:
Are there plans to move this into the ManageIQ namespace?

Reply:
hi @bdunne - yes. We will need an MIQ namespace. Please feel free @bdunne  @ZitaNemeckova to get started on one and let me know if I can help (if you have permissions). I'm fine w/ naming as you feel is appropriate.

Reply:
@Fryguy mentioned today that we do not need to pull this into ManageIQ.  Looking for clarification on this point.

Reply:
I mentioned that I'm ok with merging for now even if it lives outside of the ManageIQ namespace.  When we release it, I expect it in the ManageIQ namespace following the standards around other ManageIQ repos.  It will likely have to be renamed a little bit as well.